rebasehelper/build_log_hook.py

```diff
@@ -64,7 +64,10 @@ def run(self, spec_file, rebase_spec_file, **kwargs):
 
         """
         if not kwargs.get('non_interactive'):
+            blacklist = kwargs.get('build_log_hook_blacklist', [])
             for name, build_log_hook in six.iteritems(self.build_log_hooks):
+                if  name in blacklist:
```

Comment:
Extra space here.

Reply:
Ok, it is fixed in one of the following commits, but it should be fixed in this one.